xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/informationinline.vm

```diff
@@ -20,6 +21,19 @@
         #end
       </dd>
     </dl>
+    <dl>
```

Comment:
Do you really need to open a new definition list instead of adding the ``dt``/``dd`` to the previous list?

Reply:
I guess I'm not. I'll remove it.